Example/StripeConnectExample/StripeConnectExample/Info.plist

```diff
@@ -8,53 +8,53 @@
 	<false/>
 	<key>CFBundleLocalizations</key>
 	<array>
-		<string>bg-BG</string>
-		<string>zh-Hans</string>
-		<string>zh-Hant-HK</string>
-		<string>zh-Hant-TW</string>
-		<string>hr-HR</string>
-		<string>cs-CZ</string>
-		<string>da-DK</string>
-		<string>nl-NL</string>
-		<string>en-AU</string>
-		<string>en-IN</string>
-		<string>en-IE</string>
-		<string>en-NZ</string>
-		<string>en-SG</string>
-		<string>en-GB</string>
-		<string>en-US</string>
-		<string>et-EE</string>
-		<string>fil-PH</string>
-		<string>fi-FI</string>
-		<string>fr-CA</string>
-		<string>fr-FR</string>
-		<string>de-DE</string>
-		<string>el-GR</string>
-		<string>hu-HU</string>
-		<string>id-ID</string>
-		<string>it-IT</string>
-		<string>ja-JP</string>
-		<string>ko-KR</string>
-		<string>lv-LV</string>
-		<string>lt-LT</string>
-		<string>ms-MY</string>
-		<string>mt-MT</string>
-		<string>nb-NO</string>
-		<string>pl-PL</string>
-		<string>pt-BR</string>
-		<string>pt-PT</string>
-		<string>ro-RO</string>
-		<string>sk-SK</string>
-		<string>sl-SI</string>
-		<string>es-AR</string>
-		<string>es-BR</string>
-		<string>es-419</string>
-		<string>es-MX</string>
-		<string>es-ES</string>
-		<string>sv-SE</string>
-		<string>th-TH</string>
-		<string>tr-TR</string>
-		<string>vi-VN</string>
+		<string>bg_BG</string>
+		<string>zh_Hans</string>
+		<string>zh-Hant_HK</string>
+		<string>zh-Hant_TW</string>
+		<string>hr_HR</string>
```

Comment:
Updated these to use the apple Locale IDs, although I don't think it makes a difference
https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html#//apple_ref/doc/uid/10000171i-CH15-SW9